### DISCUSSION TITLE
New component: Tabs

### DISCUSSION THREAD

**Discussion by iharshr** [heroui-inc/heroui#316] [Feature requests]
tabs would be great addition

something like this  https://ant.design/components/tabs/

**Comment by jrgarciadev** [comment]
Hi @tetrex, this component is coming! 🚀

**Reply by iharshr** [reply]
hey @jrgarciadev this component was removed from planning board , any specific reason ?

**Reply by jrgarciadev** [reply]
Hey @tetrex I've moved all the feature requests to Github Discussions, it's still on the roadmap 🙌🏻

**Reply by Akash187** [reply]
Hi @jrgarciadev 
Any update on tabs?

**Reply by jrgarciadev** [reply]
Hey @tetrex we hope to add it after the `Autocomplete` & `Select` components

**Comment by wingkwong** [comment]
https://www.heroui.com/docs/components/tabs
